student.rb

```diff
@@ -9,4 +9,9 @@ def initialize(age, classroom, name = 'Unknown', parent_permission: true)
   def play_hooky
     '¯\\_(ツ)_/¯'
   end
+
+  def add_classroom(classroom)
+    @classroom = classroom
+    classroom.students << self

```

Comment:
- [Optional] Consider checking if the student already exists in the classroom before pushing. Use 
`classroom.students.push(self) unless classroom.students.include?(self)`.

Reply:
Fixed.